### ISSUE TITLE
The <img> element without the "alt" contains the "aria-label" attribute

### ISSUE THREAD

**Issue by sarasuri** [w3c/wcag#2963]
Hi,

We are using a tool to create Web forms. The tool uses the “aria-label” attribute to display the alt text for an image inserted in the forms within the <img> element. Therefore the <img> element doesn’t contain the “alt” attribute. For example, <img role="img" title="Testing" aria-label="Testing" …
In this case, does it meet the requirements for WCAG SC - 1.1.1?
Or the <img> element must have the "alt" attribute regardless of "aria-label" attribute?

Thanks

**Comment by patrickhlauke** [comment]
Yes `aria-label` is unorthodox but fine.

**Comment by sarasuri** [comment]
Thanks @patrickhlauke 
Ok, I understand it is fine even it is not the standard way of providing the alt text. However, it fails for the WCAG SC -4.1.1 as the W3C Validator reports the following errors. Therefore, in order to fix those errors, the <img> element must have the "alt" attribute. Is my understating correct?

![image](https://user-images.githubusercontent.com/12256067/214399422-b6c9af19-2045-4e0a-824c-c978c4a0d472.png)

![image](https://user-images.githubusercontent.com/12256067/214399501-d9e31757-f238-42d5-8dc4-cbb9096b61ea.png)
